bom/application/pom.xml

```diff
@@ -4566,16 +4566,6 @@
                 <artifactId>smallrye-connector-attribute-processor</artifactId>
                 <version>${smallrye-reactive-messaging.version}</version>
             </dependency>
-            <dependency>
-                <groupId>io.quarkus</groupId>
-                <artifactId>quarkus-reactive-messaging-http</artifactId>
-                <version>${project.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>io.quarkus</groupId>
-                <artifactId>quarkus-reactive-messaging-http-deployment</artifactId>
-                <version>${project.version}</version>
-            </dependency>
```

Comment:
Please move them with the others in the `Relocations` section further in the BOM rather than removing them.